docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -0,0 +1,2028 @@
+      "              'status': 'draft',\n",
+      "              'type': 'Extension',\n",
+      "              'url': 'http://fhir.kids-first.io/StructureDefinition/patient-race',\n",
+      "              'version': '4.0.0'},\n",
+      " 'status_code': 201}\n",
+      "POST http://hapi.fhir.org/baseR4/StructureDefinition, patient_race\n",
+      "DELETE http://fhir.kids-first.io/StructureDefinition/Patient\n",
+      "{'request_url': 'http://hapi.fhir.org/baseR4/StructureDefinition',\n",
+      " 'response': {'abstract': False,\n",
+      "              'baseDefinition': 'http://hl7.org/fhir/StructureDefinition/Patient',\n",
+      "              'derivation': 'constraint',\n",
+      "              'description': 'The individual human or other organism.',\n",
+      "              'differential': {'element': [{'id': 'Patient', 'path': 'Patient'},\n",
+      "                                           {'id': 'Patient.identifier',\n",
+      "                                            'mustSupport': True,\n",
+      "                                            'path': 'Patient.identifier'},\n",
+      "                                           {'id': 'Patient.name',\n",
+      "                                            'max': '0',\n",
+      "                                            'path': 'Patient.name'},\n",
+      "                                           {'id': 'Patient.telecom',\n",
+      "                                            'max': '0',\n",
+      "                                            'path': 'Patient.telecom'},\n",
+      "                                           {'id': 'Patient.address',\n",
+      "                                            'max': '0',\n",
+      "                                            'path': 'Patient.address'},\n",
+      "                                           {'id': 'Patient.photo',\n",
+      "                                            'max': '0',\n",
+      "                                            'path': 'Patient.photo'},\n",
+      "                                           {'id': 'Patient.contact',\n",
+      "                                            'max': '0',\n",
+      "                                            'path': 'Patient.contact'},\n",
+      "                                           {'id': 'Patient.gender',\n",
+      "                                            'min': 1,\n",
+      "                                            'mustSupport': True,\n",
+      "                                            'path': 'Patient.gender'},\n",
+      "                                           {'id': 'Patient.extension',\n",
+      "                                            'path': 'Patient.extension',\n",
+      "                                            'slicing': {'discriminator': [{'path': 'url',\n",
+      "                                                                           'type': 'value'}],\n",
+      "         
```

Comment:
Wondering if we should use a different example since we removed `name` from the Patient model earlier.